corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -112,15 +112,25 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
             <div>
                 <!-- ko if: help() -->
                 <a
-                    class="help-popover"
-                    tabindex="0"
-                    role="button"
-                    data-trigger="focus"
-                    data-placement="top"
-                    data-toggle="popover"
-                    data-bind="attr: { 'data-content': help }">
+                    class="help-text-trigger"
+                    role="button">
                     <i class="fa fa-info-circle"></i>
                 </a>
+                <div id="help-text-modal" class="modal fade" role="dialog">
+                    <div class="modal-dialog">
+                        <div class="modal-content">
+                            <div class="modal-body">
+                                <p align="left" data-bind="text: help"></p>
+                            </div>
+                            <div class="modal-footer">
+                                <!-- This needs to be a <span> instead of a <button> because (inexplicably) making it
+                                a button causes it to get clicked automatically when the modal opens, thus closing
+                                the modal immediately after it opens. We have yet to figure out why this is so. -->
+                                <span class="btn btn-default help-modal-close" data-dismiss="modal">Close</span>
```

Comment:
should be translated

Reply:
@orangejenny From looking at other templates, I assume you mean to use this pattern?:

`{% trans 'some text' %}`

The reason I ask is because in this particular file, it doesn't look like any other display strings are using that pattern (they're all just hard-coded). Not sure if there's a reason for that or just oversight. Also, am I intended to actually add the translations myself? I assume not but just not sure how this works in HQ.

Reply:
Yep, `trans` for python templates (`gettext` translates JavaScript strings, it's used elsewhere in this file). You don't add translations, there's a process that picks up the strings and puts them in transifex. The whole file should be translated, it's an oversight that it's mostly untranslated.

Reply:
@orangejenny Ok cool thanks. PR is updated